libnymea-modbus/tools/connectiontool/modbusrtu.py

```diff
@@ -63,7 +65,7 @@ def writePropertyGetSetMethodImplementationsRtu(fileDescriptor, className, regis
             elif registerDefinition['registerType'] == 'coils':
                 writeLine(fileDescriptor, '    return m_modbusRtuMaster->writeCoils(m_slaveId, %s, values);' % (registerDefinition['address']))
             else:
-                print('Error: invalid register type for writing.')
+                logger.warning('Error: invalid register type for writing.')
```

Comment:
Isn't there something like `logger = logging.getLogger('modbus-tools')` missing? I'm not sure, just asking as I noticed it being there in other files, but not all of them.

Reply:
it's in the common file globaly defined, there might be better ways to use in the entire application, I just made it work somehow for me.